docs/updating/update-to-43.md

```diff
@@ -16,8 +16,111 @@ modified_at: 2024-07-31
+
+Alongside with the configuration changes we decided to switch the default `auto_pagination` option to `false`. This option was only used with the pagination feature. When it is enabled the breaks in the pagination force the page breaks in the exported Word document. However, browser engines and Microsoft Word differ significantly. Because of that we decided to turn this option to `false`. If you still want to enforce breaks from pagination, change it in the configuration to `true`.
+
+### Typing bug fixes
+
+This release brings a couple of improvements to typing in the editor.
+
+* Android IME: multiple issues related to duplicated characters, reverse writing effects, etc. ([#13994](https://github.com/ckeditor/ckeditor5/issues/13994), [#14707](https://github.com/ckeditor/ckeditor5/issues/14707), [#13850](https://github.com/ckeditor/ckeditor5/issues/13850), [#13693](https://github.com/ckeditor/ckeditor5/issues/13693), [#14567](https://github.com/ckeditor/ckeditor5/issues/14567), [#11569](https://github.com/ckeditor/ckeditor5/issues/11569)).
+* Safari: the reverse typing effect after the focus change ([#14702](https://github.com/ckeditor/ckeditor5/issues/14702)).
+
+These fixes do not require migration, but typing is a crucial area for the editor, and we want to call them out explicitly.
+
+### Update of global names in the UMD builds
+
+We have decided to change the global names for the `ckeditor5` and `ckeditor5-premium-features` packages in the UMD builds to `CKEDITOR` and `CKEDITOR_PREMIUM_FEATURES` respectively.
+
+### React and Vue integrations updates
+
+We have released new major versions of the React and Vue integrations. In both of them, we have migrated to JavaScript modules (ESM) and rewritten large parts of their codebases to support the latest versions of these frameworks and to follow the latest recommendations for writing the components.
+
+We strongly recommend that you read the release highlights to update to the latest versions:
+
+- [Release highlights for React integration](https://github.com/ckeditor/ckeditor5-react/releases/tag/v9.0.0).
+- [Release highlights for Vue integration](https://github.com/ckeditor/ckeditor5-vue/releases/tag/v7.0.0).
+
+### Updates to the Package Generator
+
+In the last major release of CKEditor&nbsp;5 we introduced new installation methods. Since then we have received a lot of feedback from you. Thanks to this, we identified a number of minor issues fixed in versions 42.0.1 and 42.0.2. While these fixes did not require any changes in the application, they do require some changes in the custom plugins created with the Package Generator.
+
+You can skip this section if you have not yet read the [Migrating custom plugins](https://ckeditor.com/docs/ckeditor5/latest/updating/nim-migration/custom-plugins.html) guide. However, if you have, follow the steps below to apply these fixes to your plugin.
+
+1. Update all packages starting with `@ckeditor/ckeditor5-dev-` to version `^42.0.0`.
```

Comment:
Is the version right here?

Reply:
Copied from @filipsobol's notes. @pszczesniak?

Reply:
Yes `42.0.0` is the correct version for `ckeditor5-dev-*` packages.